go.mod

```diff
@@ -46,7 +47,8 @@ require (
 	github.com/duosecurity/duo_api_golang v0.0.0-20190308151101-6c680f768e74 // indirect
 	github.com/elazarl/go-bindata-assetfs v1.0.0 // indirect
 	github.com/emicklei/go-restful v2.9.3+incompatible
-	github.com/envoyproxy/go-control-plane v0.8.6
+	github.com/envoyproxy/go-control-plane v0.8.7-0.20190821215049-f062b07a671a
```

Comment:
LGTM. I checked that this points to a version of Envoy that is just a couple days older than the Envoy version used in istio/proxy.